test/integration/pubsub/test_pubsub.rb

```diff
@@ -0,0 +1,153 @@
+require "helpers/integration_test_helper"
+require "securerandom"
+require "base64"
+
+class TestPubsubService < FogIntegrationTest
+  @@client = Fog::Google::Pubsub.new
```

Comment:
I saw this error with rubocop and went back and forth on this.

I think the current code is easier to read, especially since it already has code that runs at class load time (specifically the minitest after_run code). I could factor it out to a class variable but given that the client is immutable (please correct me if i'm wrong) I'm not sure what gain we will get.

Also the above argument obviously only applies to test code; I wouldn't write this syntax in library code.